internal/hcl/parser.go

```diff
@@ -397,9 +397,28 @@ func (p *Parser) ParseDirectory() (m *Module, err error) {
 		nil,
 	)
 
-	root, err := evaluator.Run()
-	if err != nil {
-		return m, err
+	var root *Module
+
+	// Graph evaluation
+	if os.Getenv("INFRACOST_GRAPH_EVALUATOR") == "true" {
```

Comment:
@hugorut should we move all this logic inside `evaluator.Run`?

This would also fix the (not-important) issue that we're not seeing the `⠴ Evaluating Terraform directory` spinner when using the graph evaluator.